DOCSP-27098 File location ref page (#38)
mongodb/docs-relational-migrator: source/installation/file-location.txt

.. _file-location:

==================================
Relational Migrator File Locations
==================================

.. default-domain:: mongodb

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 1
   :class: singlecol

> Review the location of files needed when working with Relational Migrator. 

Reviewer: This intro doesn't adequately explain why users might care about this. In most cases people won't need this info, but it can be useful in a few situations:
 - Deploying additional JDBC drivers
 - Modifying application configuration files
 - Reviewing log files

Their location varies according to the OS you are working with. 

Mac
---

The file locations for the Mac OS:

- **Configuration file**
  
  ``~/Library/Application Support/MongoDB/Relational Migrator/user.properties``

> - **Driver downloads**

Reviewer: This term is unclear. Maybe "Downloaded JDBC Drivers"

>   ``~/Library/Application Support/MongoDB/Relational Migrator/Drivers``

Reviewer: This is incorrect; tilde needs to be removed. This difference is subtle and should be emphasised, i.e. the config and logs are under the user's home folder, while the drivers are shared for all users.

Author: Done.

> - **Log information**

Reviewer: "Log files" would be better

Author: Done.

  
  ``~/Library/Application Support/MongoDB/Relational Migrator/Logs/migrator.log``

Windows
-------

The file locations for the Windows OS:

- **Configuration file**

>   ``c:\Users\<username>\AppData\Local\MongoDB\Relational Migrator\Data\user.properties``

Reviewer: This path is correct, but in staging this is path is too long to fully display without horizontal scrolling. Is there a better way of showing this (wrapping on one of the spaces would be preferable to horizontal scrolling).

Author: I tried breaking up the line on the spacing but it unfortunately renders the same.

.. _file-location:

==================================
Relational Migrator File Locations
==================================

.. default-domain:: mongodb

.. contents:: On this page
   :local:
   :backlinks: none
   :depth: 1
   :class: singlecol

> The following page provides the location of files needed when working with 

Reviewer 2: (wording tweak)
If you can, it's a good idea to avoid introductory phrases like "This page ..." or "The following information includes ... " The reasoning here is just that these tend to be throw away phrases. You can often tighten up the intro by omitting them. In this case you could say something like:

```suggestion
Review the location of files needed when working with 
```

> Relational Migrator. The location of the files varies according to the OS 

Reviewer 2: (wording -- minimalism)
Tweaking fit a bit better with the rewrite to the first sentence and for improved minimalism. 

```suggestion
Relational Migrator. Their location varies according to the OS 
```

you are working with. 

.. note::

> 

Reviewer 2: (question)
Is this note necessary, since we didn't GA until 1.1.2?

    This information is correct for Relational Migrator v1.0.35 and higher (earlier 
    versions store files in different locations).

Mac
---

> 

Reviewer 2: (praise)
Thanks for working through the table issue! I think this list renders with a better look and feel than the table.

The file locations for the Mac OS:

> - :guilabel:`Configuration file`

Reviewer 2: (markup)
You could just go with bold here, since :guilabel: should technically reference a UI element.

  
  ``~/Library/Application Support/MongoDB/Relational Migrator/user.properties``

- :guilabel:`Driver downloads`

>   ``/Library/Application Support/MongoDB/Relational Migrator/Drivers``

Reviewer 2: (question)
Is a ~ part of the file path here as well?

Author: Yes, thank you!
